src/replicache.ts

```diff
@@ -309,10 +308,10 @@ export default class Replicache implements ReadTransaction {
       }
     }
 
-    const {batchPushInfo} = {...syncInfo};
+    const {batchPushInfo, clientViewInfo} = syncInfo;
```

Comment:
@aboodman You changed `syncInfo` to `{...syncInfo}` in https://github.com/rocicorp/replicache-sdk-js/commit/206636c84d9882450cd0ec7eee396e46f0175ed2#diff-8cb1c873134ee240726c9258ef1aa053R277

Based on the type info (both in TS and in Rust) syncInfo cannot be `null`/`undefined`. Maybe there was a time window when it wasn't fully implemented?

Reply:
I think I was guessing.

Reply:
I wasn't sure which fields could be null or not.